revisions.py

```diff
@@ -283,7 +287,10 @@ def resource_modified_post_revision(ctx, resource, zone, path):
                                                       pathutil.dirname(path), pathutil.basename(path), revision_avu_name),
                                                   offset=0, limit=1, output=genquery.AS_LIST))) > 0
 
-        if not already_has_avu:
+        # Check that size is not too big to make a revision
+        data_size = data_object.size(ctx, path)
+
+        if int(data_size) <= constants.UUMAXREVISIONSIZE and not already_has_avu:
```

Comment:
I think the size check can now be removed here, since it is also performed in `revision_eligible`, right?

Reply:
Ah yes, thanks! I've corrected that now.